README.md

```diff
@@ -1 +1,14 @@
-# g-review
\ No newline at end of file
+G-review
```

Comment:
It's great that you've added a readme properly now, I would look at some formatting -> https://www.markdownguide.org/cheat-sheet/